### PR TITLE
FPv2: move car-specific configuration into interfaces

### PR DESCRIPTION
**Pull request by sshane**

### CODE REVIEW

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -129,12 +132,47 @@ class ChryslerCarInfo(CarInfo):
...
+  p16(uds.DATA_IDENTIFIER_TYPE.SYSTEM_SUPPLIER_ECU_SOFTWARE_NUMBER)
+CHRYSLER_SOFTWARE_VERSION_RESPONSE = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER + 0x40]) + \
+  p16(uds.DATA_IDENTIFIER_TYPE.SYSTEM_SUPPLIER_ECU_SOFTWARE_NUMBER)
+
+CHRYSLER_RX_OFFSET = -0x280
+
+FPV2_CONFIG = Fpv2Config(
+  requests=[
+    Request(
+      "chrysler",
```
Comment (adeebshihadeh) [inline selfdrive/car/chrysler/values.py:150]: should be able to remove the brand names now

File: release/files_common [hunk 2]
```diff
@@ -101,6 +101,7 @@ selfdrive/car/interfaces.py
 selfdrive/car/vin.py
 selfdrive/car/disable_ecu.py
 selfdrive/car/fw_versions.py
+selfdrive/car/fpv2_definitions.py
```
Comment (adeebshihadeh) [inline release/files_common:104]: let's refer to fpv2 as fw query in the code for consistency

File: selfdrive/car/body/values.py [hunk 3]
```diff
@@ -3,24 +3,39 @@
 from cereal import car
 from selfdrive.car import dbc_dict
 from selfdrive.car.docs_definitions import CarInfo
+from selfdrive.car.fpv2_definitions import Fpv2Config, Request, StandardQueries as SQ
```
Comment (adeebshihadeh) [inline selfdrive/car/body/values.py:6]: I generally like to rename imports judiciously and wouldn't do it in this case, but up to you

**Review by adeebshihadeh — COMMENTED** [review]
I'd like to follow this up (in another PR) by moving the addresses to query and the addr -> ECU mapping into the FW query config

File: selfdrive/car/ford/values.py [hunk 4]
```diff
@@ -61,6 +63,26 @@ class FordCarInfo(CarInfo):
   CAR.FOCUS_MK4: FordCarInfo("Ford Focus EU 2019", "Driver Assistance Pack"),
 }
 
+FORD_VERSION_REQUEST = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER]) + \
+  p16(uds.DATA_IDENTIFIER_TYPE.VEHICLE_MANUFACTURER_ECU_SOFTWARE_NUMBER)
+FORD_VERSION_RESPONSE = bytes([uds.SERVICE_TYPE.READ_DATA_BY_IDENTIFIER + 0x40]) + \
+  p16(uds.DATA_IDENTIFIER_TYPE.VEHICLE_MANUFACTURER_ECU_SOFTWARE_NUMBER)
```
Comment (incognitojam) [inline selfdrive/car/ford/values.py:69]: Looks like Mazda, Nissan and Ford all use the same request/response, could we add those to StdQueries?

**Review by incognitojam — APPROVED** [review]

**Pull request merged**